### ISSUE TITLE
Terraform apply crashes when specifying 'enabled_cloudwatch_logs_exports' & 'restore_to_point_in_time' in aws_rds_cluster

### ISSUE THREAD

**Issue by vqian-guidewire** [hashicorp/terraform-provider-aws#16466]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Terraform CLI and Terraform AWS Provider Version
Terraform 0.13.5  
Aws provider version: 3.16.0

### Affected Resource(s)

* aws_rds_cluster

### Terraform Configuration Files

```hcl
resource "aws_rds_cluster" "this" {
  cluster_identifier                                = var.name
  db_cluster_parameter_group_name = var.db_cluster_parameter_group_name
  db_subnet_group_name                    = var.db_subnet_group_name
  vpc_security_group_ids                     = var.vpc_security_group_ids

  # specifying 'enabled_cloudwatch_logs_exports' causes 'rpc error: code = Unavailable desc = transport is closing' error
  enabled_cloudwatch_logs_exports = var.enabled_cloudwatch_logs_exports

  restore_to_point_in_time {
    source_cluster_identifier      = var.pitr_source_cluster_identifier
    restore_type                          = var.pitr_restore_type
    use_latest_restorable_time  = var.pitr_use_latest_restorable_time
    restore_to_time                     = var.pitr_restore_to_time
  }

}
```

### Panic Output

https://gist.github.com/vqian-guidewire/1097f0ed86410a86fca4c1cf2f53b424

### Expected Behavior

Terraform is not crashing, and point in time recover should work, and 'enabled_cloudwatch_logs_exports' should take effect.

### Actual Behavior

Terraform crashing

### Steps to Reproduce

1. `terraform apply` by providing necessary variables.

**Comment by ewbankkit** [comment]
```
2020-11-27T10:56:00.118-0500 [DEBUG] plugin.terraform-provider-aws_v3.18.0_x5: 2020/11/27 10:56:00 [DEBUG] setting computed for "availability_zones" from ComputedKeys
2020-11-27T10:56:00.118-0500 [DEBUG] plugin.terraform-provider-aws_v3.18.0_x5: 2020/11/27 10:56:00 [DEBUG] setting computed for "cluster_members" from ComputedKeys
2020-11-27T10:56:00.121-0500 [DEBUG] plugin.terraform-provider-aws_v3.18.0_x5: panic: interface conversion: interface {} is *schema.Set, not []interface {}
2020-11-27T10:56:00.121-0500 [DEBUG] plugin.terraform-provider-aws_v3.18.0_x5: 
2020-11-27T10:56:00.121-0500 [DEBUG] plugin.terraform-provider-aws_v3.18.0_x5: goroutine 172 [running]:
2020-11-27T10:56:00.122-0500 [DEBUG] plugin.terraform-provider-aws_v3.18.0_x5: github.com/terraform-providers/terraform-provider-aws/aws.resourceAwsRDSClusterCreate(0xc001a95000, 0x61b9460, 0xc0012a5700, 0x1, 0xffffffffffffffff)
2020-11-27T10:56:00.122-0500 [DEBUG] plugin.terraform-provider-aws_v3.18.0_x5: 	/opt/teamcity-agent/work/5d79fe75d4460a2f/src/github.com/hashicorp/terraform-provider-aws/aws/resource_aws_rds_cluster.go:750 +0x6ea1
2020-11-27T10:56:00.122-0500 [DEBUG] plugin.terraform-provider-aws_v3.18.0_x5: github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*Resource).create(0xc000b34210, 0x7a00c60, 0xc001714780, 0xc001a95000, 0x61b9460, 0xc0012a5700, 0x0, 0x0, 0x0)
2020-11-27T10:56:00.122-0500 [DEBUG] plugin.terraform-provider-aws_v3.18.0_x5: 	/opt/teamcity-agent/work/5d79fe75d4460a2f/pkg/mod/github.com/hashicorp/terraform-plugin-sdk/v2@v2.3.0/helper/schema/resource.go:268 +0x88
2020-11-27T10:56:00.122-0500 [DEBUG] plugin.terraform-provider-aws_v3.18.0_x5: github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*Resource).Apply(0xc000b34210, 0x7a00c60, 0xc001714780, 0xc000c14bd0, 0xc001748420, 0x61b9460, 0xc0012a5700, 0x0, 0x0, 0x0, ...)
2020-11-27T10:56:00.122-0500 [DEBUG] plugin.terraform-provider-aws_v3.18.0_x5: 	/opt/teamcity-agent/work/5d79fe75d4460a2f/pkg/mod/github.com/hashicorp/terraform-plugin-sdk/v2@v2.3.0/helper/schema/resource.go:386 +0x681
2020-11-27T10:56:00.122-0500 [DEBUG] plugin.terraform-provider-aws_v3.18.0_x5: github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*GRPCProviderServer).ApplyResourceChange(0xc0010da920, 0x7a00c60, 0xc001714780, 0xc001664960, 0xc001714780, 0xc001712f00, 0x7a2f580)
2020-11-27T10:56:00.122-0500 [DEBUG] plugin.terraform-provider-aws_v3.18.0_x5: 	/opt/teamcity-agent/work/5d79fe75d4460a2f/pkg/mod/github.com/hashicorp/terraform-plugin-sdk/v2@v2.3.0/helper/schema/grpc_provider.go:951 +0x8b2
2020-11-27T10:56:00.122-0500 [DEBUG] plugin.terraform-provider-aws_v3.18.0_x5: github.com/hashicorp/terraform-plugin-go/tfprotov5/server.(*server).ApplyResourceChange(0xc00170edc0, 0x7a00c60, 0xc001714780, 0xc000c14a80, 0xc00170edc0, 0xc001701e60, 0xc001891ba0)
2020-11-27T10:56:00.122-0500 [DEBUG] plugin.terraform-provider-aws_v3.18.0_x5: 	/opt/teamcity-agent/work/5d79fe75d4460a2f/pkg/mod/github.com/hashicorp/terraform-plugin-go@v0.1.0/tfprotov5/server/server.go:331 +0xac
2020-11-27T10:56:00.122-0500 [DEBUG] plugin.terraform-provider-aws_v3.18.0_x5: github.com/hashicorp/terraform-plugin-go/tfprotov5/internal/tfplugin5._Provider_ApplyResourceChange_Handler(0x6c48d20, 0xc00170edc0, 0x7a00d20, 0xc001701e60, 0xc001712f60, 0x0, 0x7a00d20, 0xc001701e60, 0xc00167f500, 0xa4d)
2020-11-27T10:56:00.122-0500 [DEBUG] plugin.terraform-provider-aws_v3.18.0_x5: 	/opt/teamcity-agent/work/5d79fe75d4460a2f/pkg/mod/github.com/hashicorp/terraform-plugin-go@v0.1.0/tfprotov5/internal/tfplugin5/tfplugin5_grpc.pb.go:380 +0x217
2020-11-27T10:56:00.122-0500 [DEBUG] plugin.terraform-provider-aws_v3.18.0_x5: google.golang.org/grpc.(*Server).processUnaryRPC(0xc0000248c0, 0x7a25740, 0xc0004ba480, 0xc0018d4600, 0xc000f9dc80, 0xb1bbf20, 0x0, 0x0, 0x0)
2020-11-27T10:56:00.122-0500 [DEBUG] plugin.terraform-provider-aws_v3.18.0_x5: 	/opt/teamcity-agent/work/5d79fe75d4460a2f/pkg/mod/google.golang.org/grpc@v1.32.0/server.go:1194 +0x50a
2020-11-27T10:56:00.122-0500 [DEBUG] plugin.terraform-provider-aws_v3.18.0_x5: google.golang.org/grpc.(*Server).handleStream(0xc0000248c0, 0x7a25740, 0xc0004ba480, 0xc0018d4600, 0x0)
2020-11-27T10:56:00.122-0500 [DEBUG] plugin.terraform-provider-aws_v3.18.0_x5: 	/opt/teamcity-agent/work/5d79fe75d4460a2f/pkg/mod/google.golang.org/grpc@v1.32.0/server.go:1517 +0xcfd
2020-11-27T10:56:00.122-0500 [DEBUG] plugin.terraform-provider-aws_v3.18.0_x5: google.golang.org/grpc.(*Server).serveStreams.func1.2(0xc000c40c50, 0xc0000248c0, 0x7a25740, 0xc0004ba480, 0xc0018d4600)
2020-11-27T10:56:00.122-0500 [DEBUG] plugin.terraform-provider-aws_v3.18.0_x5: 	/opt/teamcity-agent/work/5d79fe75d4460a2f/pkg/mod/google.golang.org/grpc@v1.32.0/server.go:859 +0xa1
2020-11-27T10:56:00.122-0500 [DEBUG] plugin.terraform-provider-aws_v3.18.0_x5: created by google.golang.org/grpc.(*Server).serveStreams.func1
2020-11-27T10:56:00.122-0500 [DEBUG] plugin.terraform-provider-aws_v3.18.0_x5: 	/opt/teamcity-agent/work/5d79fe75d4460a2f/pkg/mod/google.golang.org/grpc@v1.32.0/server.go:857 +0x204
```

**Comment by anGie44** [comment]
Hi @vqian-guidewire, the fix has been merged and will release with `v3.19.0` of the Terraform AWS Provider, likely out this Thursday 👍
